content/en/apps/guides/data/on-boarding-data-integrity.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Best Practices for Data Integrity During On-Boarding"
+linkTitle: "Data Integrity in On-Boarding"
+weight: 15
+description: >
+    How to ensure training data ends up in training and production data ends up production.
+
+---
+
+{{% pageinfo %}}
+This best practices guide should not be followed verbatim.  Instead, it is important to review each suggestion and assess if your deployment can benefit from it.
+{{% /pageinfo %}}
+
+## Introduction
+
+When on-boarding CHWs or refreshing their training, it is important to both enable them to learn how to use the CHT using a real mobile device with a real CHT Android application on the exact same configuration as production will use. This will give them the confidence to correctly and safely use the app to do their first household visits. This guide assumes that CHWs will have both a training and production app installed at the same time, or the CHW will be instructed to go install and log into the production app when training has been completed.
+
+Just as important, is to ensure their training data does not end up polluting production data.
+
+On-boarding is not a one-size-fits-all process. Be sure to assess which of the items below meet your deployment's needs and implement them accordingly. Within this assessment, consider if extra set up is needed (eg couch2pg hooked up to training) to enable the size you need to fit your deployment.
+
+## Training and Documentation
+
+When training CHWs, it is critical that they understand when they're using the training instance vs. the production instance. Consider adding a step to training documentation of "Ensure you're in the training app by confirming the 'URL' in 'About' under the hamburger menu". While laborious, it is a relatively quick check which empowers the CHW to know which instance they're using:
+
+![App URL in settings](app.url.png)
+
+The URL may not be a good proxy for CHWs to know which instance they're on. Will a CHW looking at  "example.**dev**.medicmobile.org" be able to tell the difference from "example.**app**.medicmobile.org"? To aid  CHWs, consider using a [different icon for the launcher and login]({{< relref "core/guides/android/branding" >}}).
```

Comment:
It would be nice to avoid having medicmobile.org in here, so perhaps `project.dev.example.com`

Reply:
agreed! fixed in 3d1f96b .